src/jupyter_contrib_nbextensions/nbextensions/highlighter/highlighter.js

```diff
@@ -245,23 +245,15 @@ $("#higlighter_menu").css({
 
 $("#higlighter_menu")
     .on('click', function() {
-    $("#submenu").toggle();
-    if($("#menu-hgl").hasClass("fa-caret-right")){ 
-        $("#menu-hgl").switchClass("fa-caret-right","fa-caret-left") 
-    } 
-    else {
-        $("#menu-hgl").switchClass("fa-caret-left","fa-caret-right")
-    }
-    })
-.tooltip({
-        title: 'Highlight selected text',
-        placement: "bottom",     
-        trigger: "hover",
-        delay: {
-            show: 500,
-            hide: 50
+        $("#submenu").toggle();
+        if ($("#menu-hgl").hasClass("fa-caret-right")) {
```

Comment:
this `if`/`else` block looks like it could be simplified to just `$("#menu-hgl").toggleClass("fa-caret-right fa-caret-left")`?